code/WorkInProgress/AzrunStuff.dm

```diff
@@ -58,4 +58,352 @@
 			init_basic(1.0 - ( eulers ** ( -sample_interval / time_const )))
 
 
+/datum/teg_transformation_clock
+	var/obj/machinery/power/generatorTemp/generator
+	var/static/list/possible_transformations
```

Comment:
what can be in this list

Reply:
I did not know you could type lists, unless your commit yesterday. Neato